atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -106,6 +106,13 @@ def get_next_release_append(self, component, version, base_release):
 
             suffix += 1
 
+    def check_build_existence_for_explicit_release(self, component, version, release):
+        build_info = {'name': component, 'version': version, 'release': release}
+        self.log.debug('checking that the build does not exist: %s', build_info)
+        build = self.xmlrpc.getBuild(build_info)
+        if build:
+            raise RuntimeError('build already exists in Koji: {}'.format(build_info))
```

Comment:
Could we inform the build ID here?

Reply:
I did this because it is reported like that also in other methods, but if we really want I can report string: component-version-release

Reply:
I mean the koji buildID (for the existing build), wouldn't that be nice for the users to have in case one reaches this code path?

Reply:
sure